spec/honeycomb/integrations/active_support_spec.rb

```diff
@@ -173,6 +173,26 @@
         expect(this_event.data["honeycomb.nonspecific_event.honeycomb"]).to eq 1
       end
 
+      it "passes along exception information" do
```

Comment:
This was a dope test. 👍🏻 

It helped me get my brain around adding a whole new case with nothing very custom for the default handler. I'd like to push that test case to this PR's branch, if you are OK with that and have the maintainers can update this PR enabled.

Reply:
@robbkidd go for it (and I did choose that option)